### PR TITLE
feat(portex): raise "GitNotFoundError" when "git" command not found

### PR DESCRIPTION
**Pull request by linjiX**

### CODE REVIEW

**Comment by GravitiRob** [comment]
Task linked: [CU-2f8wzj6 对 builder 中的 git 命令添加详细的报错信息](https://app.clickup.com/t/2f8wzj6)

File: graviti/exception.py [hunk 1]
```diff
@@ -27,6 +27,52 @@ def __str__(self) -> str:
...
+class PortexError(GravitiException):
+    """This is the base class for custom exceptions in Graviti portex module."""
+
+
+class GitNotFoundError(PortexError):
+    """This class defines the exception for the git command not found error.
+
+    Arguments:
+       message: The error message.
+       file_not_found_error: The FileNotFoundError raised from the subprocess.run().
```
Comment (Lee-000) [inline graviti/exception.py:39]: What is this?

File: graviti/exception.py [hunk 2]
```diff
@@ -27,6 +27,52 @@ def __str__(self) -> str:
...
+
+    """
+
+    _MESSAGE = (
+        "'git' command failed, most likely due to the 'git' is not installed.\n"
+        "Please install 'git' first: https://git-scm.com/book/en/v2/Getting-Started-Installing-Git"
+    )
+
+    def __init__(self, message: str = _MESSAGE):
```
Comment (Lee-000) [inline graviti/exception.py:48]: Should we add a class attr default message to `GravitiException` and rewrite its init method?

File: graviti/exception.py (hunk 1 above)
Comment (linjiX) [reply graviti/exception.py:39]: removed

File: graviti/exception.py (hunk 2 above)
Comment (linjiX) [reply graviti/exception.py:48]: actually I think we can remove the `message` in `GravitiException`

Comment (linjiX) [reply graviti/exception.py:47]: Maybe we can add a `MessageMixin` to handle message.

**Review by Lee-000 — APPROVED** [review]

**Pull request merged**